### DISCUSSION TITLE
Filtered timestamp is probably invalid

### DISCUSSION THREAD

**Discussion by PDeeney** [PlusToolkit/PlusLib#962] [Q&A]
I'm getting the following message when trying connect. It seems that the transforms are not being received by Slicer. Any idea what might be causing this?

Filtered timestamp is probably invalid for tracker buffer item with item index=92, time=5.002. The item may have been tagged with an inaccurate timestamp, therefore it will not be recorded.| in E:\D\PTNPFTb\PlusLib\src\PlusDataCollection\vtkPlusBuffer.cxx(650)

**Comment by Sunderlandkyl** [comment]
Can you upload your config file?

**Reply by PDeeney** [reply]
[ASTMFAIPointer.txt](https://github.com/PlusToolkit/PlusLib/files/8960381/ASTMFAIPointer.txt)

**Reply by PDeeney** [reply]
Even if I try with config files that have worked fine in the past there is the same error. I've tried swapping cameras, new ethernet cables and reinstalling the latest version of plus. Maybe there is an ethernet port setting that is impacting this?

**Reply by Sunderlandkyl** [reply]
Do you know the firmware version of your fusionTrack?

**Reply by PDeeney** [reply]
App FW 1.1.8.3, Shock FW 0.0.2.5

**Reply by Sunderlandkyl** [reply]
Can you upload a full log file as well?

**Comment by PDeeney** [comment]
[062222_130335_PlusLog.txt](https://github.com/PlusToolkit/PlusLib/files/8960914/062222_130335_PlusLog.txt)

[ASTMFAIPointer - Metalv2NEW.txt](https://github.com/PlusToolkit/PlusLib/files/8960921/ASTMFAIPointer.-.Metalv2NEW.txt)

Here's the full log, and I think the attempt for this log was with this different config file

**Reply by Sunderlandkyl** [reply]
According to the SDK compatibility matrix, the version of the SDK used in Plus is too old for your firmware version.

Could you try connecting using this package that uses a newer SDK?:
https://queensuca-my.sharepoint.com/:u:/g/personal/krs2_queensu_ca/EUzyAD4PJIZPmjz-OsMPk_gBLJR2IUZs_hq36riYyuRWsg?e=2gPlDf

If it still doesn't work, could you set log level to trace, run it again and upload the log?

**Reply by PDeeney** [reply]
Sure, I won't be able to try until next week. I'll update once I can try it out.

**Reply by PDeeney** [reply]
We were able to resolve the issue. It turns out there was a discrepancy between the config file and the geometry file names. We still see the filtered timestamp but it doesn't appear to be an issue.

**Comment by PDeeney** [comment]
We were able to resolve the issue. It turns out there was a discrepancy between the config file and the geometry file names. We still see the filtered timestamp but it doesn't appear to be an issue.
